docs/rfcs/opinionated_sampling_cr.md

```diff
@@ -0,0 +1,140 @@
+
+A policy is used to define which telemetry data should be sampled. Available policies can be found in the [tailbased sampling description](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor).
+
+```yaml
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: Sampler
+metadata:
+  name: example-sampler
+spec:
+  # Policies taken into account when making a sampling decision.
+  policies:
+    - name: "retain-error-policy"
+      type: status_code,
+      status_codes: [ERROR, UNSET]
+  # RoutingKey describes how traffic to be sampled is distributed. It can be 'traceid' or 'service'.
+  # Default is 'traceid'.
+  routingKey: traceid
+  # DecisionWait defines the time to wait before making a sampling decision.
+  # Default is 30s, specified in nanoseconds (30s = 30000000000ns).
+  decision_wait: 30000000000
+  # NumTraces defines the number of traces kept in memory.
+  # Default is 5000.
+  num_traces: 5000
+  # ExpectedNewTracesPerSec defines the expected number of new traces per second.
+  # Helps allocate memory structures. Default is 5000.
+  expected_new_traces_per_sec: 5000
+  # DecisionCache defines the settings for the decision cache.
+  # This allows sampling decisions to be cached to avoid re-processing traces.
+  decision_cache:
+    # SampledCacheSize configures the amount of trace IDs to be kept in an LRU
+    # cache, persisting the "keep" decisions for traces that may have already
+    # been released from memory.
+    # By default, the size is 0 and the cache is inactive.
+    # If using, configure this as much higher than num_traces so decisions for
+    # trace IDs are kept longer than the span data for the trace.
+    sampled_cache_size: 10000
+  # Components defines the template of all requirements to configure scheduling
+  # of all components to be deployed.
+  components:
+    loadbalancer:
+      # Defines if the component is managed by the operator
+      managementState: managed
+      resources:  # Resource requests and limits
+        limits:
+          cpu: "500m"
+          memory: "512Mi"
+        requests:
+          cpu: "200m"
+          memory: "256Mi"
+      # Node selection for component placement
+      nodeSelector:
+        environment: "production"
+      # Number of load balancer replicas (optional)
+      replicas: 2
+
+    sampler:
+      # Component managed by the operator
+      managementState: managed
+      # Number of sampler replicas (optional)
+      replicas: 3
+
+  # Telemetry settings for the sampler system (currently empty).
+  telemetry:
+    # telemetry configuration goes here (e.g., serviceMonitor or spanMetrics)
+    ...
+
+  # Exporter configuration. Only OTLP exporter supported.
+  exporter:
+    # OTLP exporter endpoint
+    endpoint: "jaeger:4317"
+```
+
+## Rollout Plan
+
+1. Introduction of the CRD in v1alpha1.
+2. First controller implementation.
+3. Implementation of e2e tests.
+4. CRD becomes part of the operator bundle.
```

Comment:
Let's adjust this to include some more iteration and design changes prior to being in the main bundle.